data/migrations/20181012111720_add_projects_table.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("projects", function(tbl) {
+    tbl.increments();
+
+    tbl.string("name").notNullable();
+
+    tbl.unique("name");
```

Comment:
I would put this at the end of line 5 so it doesn't look like there is another column on this table.